judge/models/profile.py

```diff
@@ -150,6 +150,10 @@ def organization(self):
     def username(self):
         return self.user.username
 
+    @cached_property
+    def has_any_ac(self):
```

Comment:
nit: "AC" might not be the best name (as you're technically not checking for an AC, and having 100/100 doesn't necessarily imply AC)